### DISCUSSION TITLE
where can I learn pyhton oop ?

### DISCUSSION THREAD

**Discussion by SergeyArzumanyan** [community/community#112278] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

any sites, or material ?

**Comment by RobertArustamyan** [comment, marked as answer]
From youtube

**Comment by ghostinhershell** [comment]
Please see my response here: https://github.com/orgs/community/discussions/112279#discussioncomment-8765821
